data/helpers.spec.js

```diff
@@ -0,0 +1,64 @@
+const request = require('supertest');
+const db = require('../data/dbConfig');
+const helpers = require('./helpers');
+
+describe('getAll', () => {
+  it('Should return an array', async () => {
+    try {
+      const users = await helpers.getAll();
+      expect(Array.isArray(users)).toBe(true);
+    } catch (error) {
+      throw new Error(error);
+    }
+  });
+});
+
+describe('insert', () => {
+  afterEach(async () => {
+    await db('users').truncate();
+  });
+  const testUser = {
+    username: 'edreeseg',
+    email: 'edward@reeseg.com',
+    department: 'CS',
+  };
+  it('Should return an ID number', async () => {
+    try {
+      const [id] = await helpers.insert(testUser);
+      expect(typeof id).toBe('number');
+    } catch (error) {
+      throw new Error(error);
+    }
+  });
+  it('Should return null if malformed user object is sent', async () => {
+    try {
+      const response = await helpers.insert({ test: 'test' });
+      expect(response).toBe(null);
+    } catch (error) {
+      throw new Error(error);
```

Comment:
Have you tested to make sure these `catch`s work?
I'm a bit worried that they'll end up not actually throwing anything but an empty error, but I'm not sure.

Reply:
The function of the catches, from what I remember, is to ensure the test fails when it hits the catch block.  They're not likely to throw anything of use, but without them the tests would pass regardless of the `expect`.  It would be better to refactor so they display something useful.